.wp-env.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
@kraftbj can you add `.wp-env.json` to the .distignore file, so that it will not be pushed to WordPress.org?

Reply:
940b1e0